### PR TITLE
Resize uvs when growing textures.

### PR DESCRIPTION
**Pull request by pcwalton**
Fixes browser.html.

r? @glennw

### CODE REVIEW

File: src/texture_cache.rs [hunk 1]
```diff
@@ -817,7 +817,20 @@ impl TextureCache {
...
                             if item.texture_id == old_texture_id {
                                 item.texture_id = new_texture_id;
-                                item.texture_size = Size2D::new(texture_size, texture_size)
+                                let width_factor = texture_size as f32 /
+                                    item.texture_size.width as f32;
+                                let height_factor = texture_size as f32 /
+                                    item.texture_size.height as f32;
+                                item.texture_size = Size2D::new(texture_size, texture_size);
+
+                                item.uv_rect.top_left.x /= width_factor;
```
Comment (glennw) [inline src/texture_cache.rs]: @pcwalton Is this going to introduce float accuracy issues after a number of resizes? Should we directly recalculate from the pixel rect each time instead?

**Comment by pcwalton** [comment]
@glennw r?

**Pull request merged**